### ISSUE TITLE
Error parsing AU headers

### ISSUE THREAD

**Issue by healeydave** [fwestenberg/reolink_dev#152]
I have recently installed this integration and its great. 
Although it seems to work well, I noticed I'm getting a lot of these errors in the Home Assistant log which I believe are coming from the Reolink integration. I have only put one camera on at the moment to test:

Error Log Contents:
  Logger: libav.rtsp
  Source: components/stream/worker.py:240 
  First occurred: 10 February 2021, 22:58:23 (1247 occurrences) 
  Last logged: 8:57:07

  Error parsing AU headers

System Details:
 - Home Assistant version: 2021.2.2
 - Reolink camera model: RCL-410-5MP
 - Camera firmware number: V2.0.0.647_20031401

**Comment by fwestenberg** [comment]
Please update the camera firmware first. My 410 is on version v3.0.0.65_20071001

**Comment by fwestenberg** [comment]
@healeydave is it working now?

**Comment by healeydave** [comment]
It seems better but I am still getting them. I think it's something else that triggers the messages. When the system is rebooted it seems fine then when I check sometime later, the messages will be. I'm wondering if its when I try to view the camera streams on a wall mounted tablet running fully kiosk browser that starts them off now.

**Comment by badabing2005** [comment]
> 
> 
> It seems better but I am still getting them. I think it's something else that triggers the messages. When the system is rebooted it seems fine then when I check sometime later, the messages will be there. I'm wondering if its when I try to view the camera streams on a wall mounted tablet running fully kiosk browser that starts them off now.

Nah, it happens even if you don't view the camera, I think it eventually gets into a wonky state, perhaps memory leaks accumulating on each subscription call.
For me scheduled daily reboots have addressed the issue.

**Comment by healeydave** [comment]
This is happening again:

Logger: libav.rtsp
Source: components/stream/worker.py:244 
First occurred: 19:30:46 (3364 occurrences) 
Last logged: 20:06:28

Error parsing AU headers

I updated my camera to v3.0.0.136_20121100 back in February as recommended by @fwestenberg  and I thought it had fixed it.

I am running on Home Assistant core-2021.3.4

I have also noticed my camera goes unavailable and the only way to get it working again is to delete it from the integrations page and re-add the camera. Then it starts working again for about 24 hrs (if that).
